index.js

```diff
@@ -386,6 +410,122 @@ exports.getLastThreadPost = function (thread) {
   return msg
 }
 
+
+/* post revision utils */
+
+exports.getRevisions = function(ssb, thread, callback) {
+  function collectRevisions(thread, callback) {
+    // this function walks the revisions of a given thread, collecting them up
+    // asyncly
+    var deepThread = pluckRecursively(thread, 'related')[0]
```

Comment:
Why use `pluckRecursively` if you're just going to take `[0]` ? Shouldnt this just be

```js
var deepThread = thread.related
```